cmd/dao_tests/resource_test.go

```diff
@@ -0,0 +1,165 @@
+//go:build integration
+// +build integration
+
+package main
+
+import (
+	"context"
+	"testing"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	_ "github.com/RHEnVision/provisioning-backend/internal/dao/sqlx"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/RHEnVision/provisioning-backend/internal/testing/identity"
+	"github.com/stretchr/testify/assert"
+)
+
+func createPkResource() *models.PubkeyResource {
```

Comment:
All functions in this package are shared, we need to name them very verbosely to prevent conflicts.

```suggestion
func createPubkeyResourceNoop() *models.PubkeyResource {
```